Dashboard Page Navigaton Designs

dhincal opened the pull request:
# Description

In this request we created two components which are not functional. In the near future we will make them functional as well. 

While creating these two components we used tailwindcss to make our code more readable and easier to understand. But for me tailwindcss is insufficient while making animations so we didn't hesitate to use pure CSS in animations.

The components we created are:
1) Navbar (not functional atm)
2) Sidebar for page navigation (not functional atm)

## Features of Components

### Navbar Features:
- [ ] At the beginning navbar has an collapse button for interacting with Sidebar.
- [ ] In the left navbar has an text logo of the project ( PostCMS ) .
- [ ] At the middle we made an text input for users to find their own posts.
- [ ] In the far right we have to elements which are:
   1) Create button for users to create new posts.
   2) Profile picture of user.

### Sidebar Features:
- [ ] Sidebar has 5 menus 
   1) Control Panel
   2) Contents
   3) Statics
   4) Settings
   5) Send Feedback
at the moment these menus are not functional but they will be functional shortly.
- [ ] Sidebar has 2 ways to interact with
   1) By clicking the button at the top left (which stands on navbar).
   2) By hovering over to sidebar.
interacting with navbar allows us to see all the names of the menus by expanding the sidebar
- [ ] All the menus has an hover animation

dhincal committed:
Download EditorJS Packages

Downloaded the editor.js package files with npm to client file

dhincal committed:
Import EditorJS Package in index.js

dhincal committed:
Merge branch 'feature/EditorJSImpl' into develop

dhincal committed:
Add DashboardPage Navbar

Created A Navbar And Added Following Elements To Navbar:
- An icon for sidebar to open and close
- The text logo of PostCMS
- Text input helps to find posts in profile
- An simple create button
- Profile Picture in the end of navbar
Issues:
- Text input has no borders
- Text input placeholder's color does not change when in focus mode

Issues will be fixed shortly

dhincal committed:
Navbar Hotfix

Following issues have been fixed:
- Text input is available now
- We have an placeholder image now
- Some alignment issues has been fixed

dhincal committed:
Create Sidebar For Dashboard Navigation

Sidebar Features Are:
- Sidebar has 5 menus 
   1) Control Panel
   2) Contents
   3) Statics
   4) Settings
   5) Send Feedback
at the moment these menus are not functional but they will be functional shortly.
- Sidebar has 2 ways to interact with
   1) By clicking the button top left (which stands on navbar).
   2) By hovering over to sidebar.
interacting navbar allows us to see all the names of the menus by expanding the sidebar
- All the menus has an hover animation

dhincal committed:
Merge branch 'feature/DashboardPageNavDesigns' into develop

AksitArda reviewed (changes requested):
You should move the css from the dashboard.js file to new css file

dhincal committed:
Deletion Of Unnecessary Imports

dhincal committed:
Create Dashboard CSS Module 

Created an CSS Module to carry the stylesheet code to an CSS file for cleaner look in code

dhincal committed:
Import Dashboard CSS Module

Dashboard CSS Module has been imported to the file and used in the relating element's classes

dhincal committed:
Merge branch 'feature/DashboardPageNavigationDesign' into develop

dhincal committed:
Deletion Of EditorJS
